pkg/utils/parser/sql_parser/sql_parser_test.go

```diff
@@ -0,0 +1,226 @@
+			wantErr:   false,
+		}),
+		Entry("Complex query with braces and quoted values with spaces", testData{
+			qry:       `((cloud_provider = 'Value' and name = 'val ue1') and (owner = ' value2  ' or region='b' ) ) or owner=c or name=e and region LIKE '%test%'`,
+			outQry:    "((cloud_provider = ? and name = ?) and (owner = ? or region = ?)) or owner = ? or name = ? and region LIKE ?",
+			outValues: []interface{}{"Value", "val ue1", " value2  ", "b", "c", "e", "%test%"},
+			wantErr:   false,
+		}),
+		Entry("Complex query with braces and empty quoted values", testData{
+			qry:       `((cloud_provider = 'Value' and name = '') and (owner = ' value2  ' or region='' ) ) or owner=c or name=e and region LIKE '%test%'`,
+			outQry:    "((cloud_provider = ? and name = ?) and (owner = ? or region = ?)) or owner = ? or name = ? and region LIKE ?",
+			outValues: []interface{}{"Value", "", " value2  ", "", "c", "e", "%test%"},
+			wantErr:   false,
+		}),
+	)
+
+	DescribeTable("ILIKE Keyword Parsing", parserTest,
+		Entry("Complex query with braces", testData{
+			qry:       "((cloud_provider = Value and name = value1) and (owner <> value2 or region=b ) ) or owner=c or name=e and region LIKE '%test%'",
+			outQry:    "((cloud_provider = ? and name = ?) and (owner <> ? or region = ?)) or owner = ? or name = ? and region LIKE ?",
+			outValues: []interface{}{"Value", "value1", "value2", "b", "c", "e", "%test%"},
+			wantErr:   false,
+		}),
+	)
+
+	DescribeTable("JSONB Query Parsing", parserTest,
+		Entry("JSONB query", testData{
+			qry:       `manifest->'data'->'manifest'->'metadata'->'labels'->>'foo' = 'bar'`,
+			outQry:    "manifest -> 'data' -> 'manifest' -> 'metadata' -> 'labels' ->> 'foo' = ?",
+			outValues: []interface{}{"bar"},
+			wantErr:   false,
+		}),
+		Entry("Invalid JSONB query", testData{
+			qry:        `manifest->'data'->'manifest'->'metadata'->'labels'->'foo' = 'bar'`,
+			outQry:     "manifest -> 'data' -> 'manifest' -> 'metadata' -> 'labels' ->> 'foo' = ?",
+			outValues:  nil,
+			wantErr:    true,
+			errMessage: "[59] error parsing the filter: unexpected token `=`",
+		}),
+		Entry("Complex JSONB query", testData{
+			qry: `manifest->'data'->'manifest'->'metadata'->'labels'->>'foo' = 'bar' and ` +
+				`( manifest->'data'->'manifest' ->> 'foo' in ('value1', 'value2') or ` +
+				`manifest->'data'->'manifest'->>'labels' <> 'foo1')`,
+			outQry: "manifest -> 'data' -> 'manifest' -> 'metadata' -> 'labels' ->> 'foo' = ? and " +
+				"(manifest -> 'data' -> 'manifest' ->> 'foo' in( ? , ?) or " +
+				"manifest -> 'data' -> 'manifest' ->> 'labels' <> ?)",
+			outValues: []interface{}{"bar", "value1", "value2", "foo1"},
+			wantErr:   false,
+		}),
+		Entry("JSONB Query @>", testData{
+			qry:       `resources.payload -> 'data' -> 'manifests' @> '[{"metadata":{"labels":{"foo":"bar"}}}]'`,
+			outQry:    "resources.payload -> 'data' -> 'manifests' @ > ?",
```

Comment:
the outQry should be `"resources.payload -> 'data' -> 'manifests' @> ?"`. there should be have a space between `@` and `>`,

Reply:
Good catch. The issue here is that the SQLScanner treated the @ and the > as two different tokens.
I fixed that so that now `@>` is one single token. 
Let me know if that works for you!